Rupak

Rupak1017 opened the pull request:
Completed content updates requested by Amil on the Home Page, AI page, Our People1 page, Our people2 page(Amil's Story), and Publication Page. Please refer to this document for Amil's changes on 07/27/24 https://docs.google.com/document/d/1isZiE1mJKFrgvvVvrW5qFeaQ5LkImQ2QB671ezzeFrs/edit?hl=en

Content changes on home page
![image](https://github.com/user-attachments/assets/bd49b750-811c-4368-a679-92ff129130f0) 

Advisor name updation zoom out view 
![image](https://github.com/user-attachments/assets/23b25916-fc6d-4461-b98f-577d3bdcdfe6)

Alt image updation and content changes of Amil's Story.
![image](https://github.com/user-attachments/assets/ae4676f6-c789-4245-bd07-1be987b905ed)

AI Page h1 change
![image](https://github.com/user-attachments/assets/4ab03679-ff0b-4d35-816f-24f072a86cc4)

Publication page h2 change
![image](https://github.com/user-attachments/assets/004b24c8-c521-4f4e-a855-cd99d369db8c)

Rupak1017 committed:
Content Updated- Homepage

Rupak1017 committed:
Updated content of AI page

Rupak1017 committed:
Updated Advisor's List and Amil's Qualifications.

Removed Anaïs Rameau, and Richard Swartzbaugh as requested from the advisor's list.

Rupak1017 committed:
Update publications.ts

Rupak1017 committed:
Update OurPeople2.ts

Rupak1017 committed:
alt image updation

Rupak1017 committed:
Update OurPeople1.ts

ridham-virufy reviewed (commented):
Why is the Join Our team button on the publications page going to share your cough module? can you fix that error? @Rupak1017

a-wong-8 reviewed (commented):
Need to change the join out team button to correct link. And add question mark to "Interested to collaborate on our research"

a-wong-8 commented on src/i18n/en/publications.ts:
```diff
@@ -42,8 +42,8 @@ export const Publications = {
   ],
   coughSection: {
-    title: "Let's help the world and save lives by recording your cough",
+    title: "Interested to collaborate on our research",
```
question mark needed here

Rupak1017 committed:
Question mark updated

Rupak1017 committed:
Redirected Join our Team button

Redirected join our team button to job listing page and commented out previous code .

ridham-virufy reviewed (approved):
Good Work
